README.md

```diff
@@ -9,6 +9,43 @@ This repository contains *unsupported* code that can be used in conjunction with
 - code for provisioning various cloud providers (GCE, AWS, VMWare, [Openstack](https://github.com/openshift/openshift-ansible-contrib/tree/master/playbooks/provisioning/openstack) and [Red Hat Virtualization (RHV) / oVirt](https://github.com/openshift/openshift-ansible-contrib/tree/master/reference-architecture/rhv-ansible))
 - supporting scripts and playbooks for the various [reference architectures](https://github.com/openshift/openshift-ansible-contrib/tree/master/reference-architecture) Red Hat has published
 
+## Running Custom Post-Provision Actions
+
+If you'd like to run post-provision actions, one option is to create a custom playbook. Here's one example that adds additional YUM repositories:
```

Comment:
nitpick: You say _one option is to_ but then you don't introduce any other way of doing it, therefore this formulation can be a bit confusing. So either mention another way of running post-provision actions, or change the first sentence to something like this: _You can run post-provision actions by creating a custom playbook._